cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControl.java

```diff
@@ -237,6 +247,10 @@ public static CruiseControl fromCrd(
         }
     }
 
+    private boolean hasMetricsConfig() {
+        return metrics != null;
+    }
```

Comment:
Would it make sense to just inline `metrics != null` in the two places where you use this instead of creating a separate method?